config/locales/en.yml

```diff
@@ -410,7 +410,7 @@ en:
       title: "Backlog"
       text: "The backlog MUST have all stories by order of PRIORITY. Most valuable and ready-to-start stories at the top and optional or uncertain stories at the very bottom"
     in_progress:
```

Comment:
Can you change the key to `current_sprint` instead of `in_progress` too?

Reply:
At the moment we can't do it, because if we make it, we must change all files that are in_progress! But in a few days, we are going to talk about it.Tnks for the comment ((:

Reply:
I don't think it's a good idea, it'll imply in a lot of changes that are not needed right now.